### PR TITLE
cmd: various golint fixes

### PR DESCRIPTION
**Pull request by kielbarry**
Assorted fixes to golint warnings. Updated preferred capitalization for variable names and comments

### CODE REVIEW

**Comment by intizarabe** [comment]
I try to learn the codes but still not good. As i understand this is
pushing me to exec you plan right? Followed by legal warnings? Is it
correct?

File: cmd/evm/json_logger.go [hunk 1]
```diff
@@ -32,6 +32,7 @@ type JSONLogger struct {
 	cfg     *vm.LogConfig
 }
 
+// NewJSONLogger assigns values to JSONLogger struct.
```
Comment (karalabe) [inline cmd/evm/json_logger.go]: Please use:

```
// NewJSONLogger creates a new EVM tracer that prints execution steps as JSON objects
// into the provided stream.
```

File: cmd/evm/staterunner.go [hunk 2]
```diff
@@ -38,6 +38,7 @@ var stateTestCommand = cli.Command{
 	ArgsUsage: "<file>",
 }
 
+// StatetestResult values will be printed to the console by stateTestCmd.
```
Comment (karalabe) [inline cmd/evm/staterunner.go]: Please replace with:

```
// StatetestResult contains the execution status after running a state test, any
// error that might have occurred and a dump of the final state if requested.
```

**Comment by kielbarry** [comment]
Awesome, thank you for the clarification :). This should be good to go now.

**Pull request merged**